interceptors/logging/payload.go

```diff
@@ -17,7 +16,7 @@ import (
 var (
 	// JsonPbMarshaller is the marshaller used for serializing protobuf messages.
 	// If needed, this variable can be reassigned with a different marshaller with the same Marshal() signature.
-	JsonPbMarshaller JsonPbMarshaler = &jsonpb.Marshaler{}
+	JsonPbMarshaller JsonPbMarshaler = &jsonpb.MarshalOptions{}
```

Comment:
Would be nice to remove this global and make it a parameter to logging options or something. Could be another PR.

Reply:
Sure, will make an issue for it.